# Prefer using the not-equals operator (`a !== b`) over inverting the equals operator (`!(a === b)`) (`nitpick/prefer-not-equals`)

ğŸ’¼ This rule is enabled in the following configs: ğŸŒ `all`, âœ… `recommended`.

ğŸ’¡ This rule is manually fixable by [editor suggestions](https://eslint.org/docs/latest/use/core-concepts#rule-suggestions).

<!-- end auto-generated rule header -->

## Rule Details

âŒ **Incorrect** code:

```js
!(a === b);
!(a == b);

!(a === b || c);
!(d > e || a === b || c);
```

âœ… **Corrrect** code:

```js
a !== b;
a != b;

a !== b || !c;
!(d > e) || a !== b || !c;
```
